### ISSUE TITLE
Change in pywb WARC API has broken warc iterator

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#380]
```
$ ipwb index ipwb/samples/warcs/froggie.warc.gz
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/ipwb", line 11, in <module>
    load_entry_point('ipwb==0.2018.1.31.210', 'console_scripts', 'ipwb')()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 572, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2755, in load_entry_point
    return ep.load()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2408, in load
    return self.resolve()
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2414, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/ipwb/__main__.py", line 10, in <module>
    import indexer
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/ipwb/indexer.py", line 24, in <module>
    from pywb.warc.archiveiterator import DefaultRecordParser
ImportError: No module named warc.archiveiterator
```
This likely has to do with the pywb release and its use of warcio.

Mitigation plan: explicitly specify previous version of pywb to be pulled as a dependency from pip.
Migration plan: move to pywb 2.0's WARC record iteration scheme or, even better, decouple from pywb and use warcio alone. There may be other couplings to pywb preventing this.

/cc @ikreymer

**Comment by machawk1** [comment]
Mitigated in e828335a3ca2ac4cafd1376f8d1cad4a5227af00.

**Comment by machawk1** [comment]
Closed with move to warcio in 95ec3c6231a33366f16b4b92be9dde32bf8dc609.
